nntrainer/graph/network_graph.cpp

```diff
@@ -212,6 +212,15 @@ int NetworkGraph::realizeActivationType(
   layer->input_layers.clear();
   layer->input_layers.push_back(current.getName());
 
+  layer->num_outputs = current.num_outputs;
```

Comment:
add a check that the number of output layers here are 1?

Reply:
I think it is better the checking in setOutputLayer(). I will.

Reply:
in setOutputLayer also possible, but for activation, only 1 output and 1 input is supported. So, it must be ensured that when setting the output here, the number of output is 1.

Reply:
done